src/components/List/ListItem/ListItem.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './ListItem.css';
+import vectorLayer, {view, defaultStyle, hoverStyle, selectStyle} from '../../../map.js'
+
+const ListItem = (props) => (
+    <tr className="list-items" 
+    onMouseOver={() => {vectorLayer.getSource().uidIndex_[props.id].setStyle(hoverStyle)}} 
```

Comment:
użycie "_" w nazwie zmiennej lub metody najczęściej oznacza w JS pole prywatne biblioteki, do którego jej klient nie powinien się odnosić. 

znacznie bezpieczniej jest korzystać z publicznego api, które znajdziemy w dokumentacji. to co jest określone jako prywatne może w kolejnych wersjach zostać zmienione bez ostrzeżenia i utworzyć trudne w naprawie błędy w naszej aplikacji